packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -336,6 +341,39 @@ abstract class VpcBase extends Resource implements IVpc {
     };
   }
 
+  /**
+   * Adds a VPN Gateway to this VPC
+   */
+  public enableVpnGateway(options: VpnGatewayProps): void {
+    if (this.vpnGatewayId) {
+      // already enabled
```

Comment:
Feels like this should be an error.

Reply:
Done

Reply:
Although it shouldn't necessarily throw an error, since multiple calls are practically harmless

Reply:
Not if the parameters are different. Did you check?

Reply:
Fair enough. Easier to throw an error. Done